src/resolve/resolved-def.h

```diff
@@ -22,18 +22,22 @@
 
 #include <inttypes.h>
 
-#define SD_RESOLVED_DNS           (UINT64_C(1) << 0)
-#define SD_RESOLVED_LLMNR_IPV4    (UINT64_C(1) << 1)
-#define SD_RESOLVED_LLMNR_IPV6    (UINT64_C(1) << 2)
-#define SD_RESOLVED_MDNS_IPV4     (UINT64_C(1) << 3)
-#define SD_RESOLVED_MDNS_IPV6     (UINT64_C(1) << 4)
-#define SD_RESOLVED_NO_CNAME      (UINT64_C(1) << 5)
-#define SD_RESOLVED_NO_TXT        (UINT64_C(1) << 6)
-#define SD_RESOLVED_NO_ADDRESS    (UINT64_C(1) << 7)
-#define SD_RESOLVED_NO_SEARCH     (UINT64_C(1) << 8)
-#define SD_RESOLVED_AUTHENTICATED (UINT64_C(1) << 9)
-
-#define SD_RESOLVED_LLMNR         (SD_RESOLVED_LLMNR_IPV4|SD_RESOLVED_LLMNR_IPV6)
-#define SD_RESOLVED_MDNS          (SD_RESOLVED_MDNS_IPV4|SD_RESOLVED_MDNS_IPV6)
-
-#define SD_RESOLVED_PROTOCOLS_ALL (SD_RESOLVED_MDNS|SD_RESOLVED_LLMNR|SD_RESOLVED_DNS)
+#include "time-util.h"
+
+#define SD_RESOLVED_DNS                (UINT64_C(1) << 0)
+#define SD_RESOLVED_LLMNR_IPV4         (UINT64_C(1) << 1)
+#define SD_RESOLVED_LLMNR_IPV6         (UINT64_C(1) << 2)
+#define SD_RESOLVED_MDNS_IPV4          (UINT64_C(1) << 3)
+#define SD_RESOLVED_MDNS_IPV6          (UINT64_C(1) << 4)
+#define SD_RESOLVED_NO_CNAME           (UINT64_C(1) << 5)
+#define SD_RESOLVED_NO_TXT             (UINT64_C(1) << 6)
+#define SD_RESOLVED_NO_ADDRESS         (UINT64_C(1) << 7)
+#define SD_RESOLVED_NO_SEARCH          (UINT64_C(1) << 8)
+#define SD_RESOLVED_AUTHENTICATED      (UINT64_C(1) << 9)
+
+#define SD_RESOLVED_LLMNR              (SD_RESOLVED_LLMNR_IPV4|SD_RESOLVED_LLMNR_IPV6)
+#define SD_RESOLVED_MDNS               (SD_RESOLVED_MDNS_IPV4|SD_RESOLVED_MDNS_IPV6)
+
+#define SD_RESOLVED_PROTOCOLS_ALL      (SD_RESOLVED_MDNS|SD_RESOLVED_LLMNR|SD_RESOLVED_DNS)
```

Comment:
uh, no need to reindent these definitions... I mean, aligning similar stuff whith whitespace is one thing, but the flags definitions defined here have little to do with the timeout definition you are adding.

In the interest of keeping patches short, can you leave this part as is, and just add the new time definition?